docs/src/main/asciidoc/spring-cloud-config.adoc

```diff
@@ -663,6 +663,14 @@ You should see a response similar to the following:
 }
 ----
 
+The default way for a client to provide the necessary authentication to let Config Server talk to Vault is to set the X-Config-Token header.
+However, you can instead omit the header and configure the authentication in the server, by setting the same configuration properties as Spring Cloud Vault.
+The property to set is `spring.cloud.config.server.vault.authentication`.
+See the https://cloud.spring.io/spring-cloud-vault/reference/html/#vault.config.authentication[Spring Cloud Vault Reference Guide] for more detail.
+
+IMPORTANT: If you omit the X-Config-Token header and use a server property to set the authentication, the Config Server application needs an additional dependency on Spring Vault (not Spring Cloud Vault) to enable the additional authentication options.
```

Comment:
I think we can leave out the `(not Spring Cloud Vault)` part

Reply:
OK